tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./node_modules/@adonisjs/mrm-preset/_tsconfig",
+  "compilerOptions": {
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true
+  },
+  "files": [
+    "./node_modules/@adonisjs/core/build/adonis-typings/index.d.ts",
+    "./node_modules/@adonisjs/redis/build/adonis-typings/index.d.ts",
```

Comment:
Should this be excluded after this? https://github.com/Rocketseat/adonis-bull/pull/39/commits/eaaaecd3c80d080355fc743699e0efa7311a6c36